lib/ebaze_web/controllers/session_controller.ex

```diff
@@ -0,0 +1,22 @@
+defmodule EbazeWeb.SessionController do
+    use EbazeWeb, :controller
+
+    alias Ebaze.Repo
+    alias Ebaze.{Accounts, Accounts.User}
+  
+    def new(conn, _)do
+        changeset = Accounts.change_user(%User{})
```

Comment:
# of spaces is diff here

Reply:
82464befda10fd9d75eb55e34eaefeafaa10469e